testingfacade/APIs/schemas/QuestionSchema.json

```diff
@@ -0,0 +1,130 @@
+                    ],
+                    "items": {
+                        "description": "Individual answer details, the label, description and id of the relevant resource and an associated answer_id",
+                        "type": "object",
+                        "required": [
+                            "answer_id",
+                            "label",
+                            "description",
+                            "id",
+                            "answer_str"
+                        ],
+                        "properties": {
+                            "answer_id": {
+                                "type": "string",
+                                "description": "ID unique to each answer within a question. This is the value to be returned for validation."
+                            },
+                            "label": {
+                                "type": "string",
+                                "description": "Resource label as might be displayed on the NCuT"
+                            },
+                            "description": {
+                                "type": "string",
+                                "description": "Resource description as might be displayed on the NCuT"
+                            },
+                            "id": {
+                                "type": "string",
+                                "description": "Resource ID as might be displayed on the NCuT"
+                            },
+                            "answer_str": {
+                                "type": "string",
+                                "description": "String combining all the resource details for display on the testing facade"
+                            }
+                        }
+                    }
+                }
+            ]
+        },
+        "time_sent": {
+            "type": "number",
+            "description": "Time the question was sent, unix timestamp in seconds"
+        },
+        "timeout": {
+            "type": "number",
+            "description": "Time in seconds the test suite will wait to receive an answer to a particular question",
+            "default": 600
+        },
+        "answer_uri": {
+            "type": "string",
+            "description": "URL of the test suite API endpoint to send the POST request with answers"
+        },
+        "metadata": {
+            "type": ["object", "null"],
+            "description": "Extra information to assist fully automated testing where an action is required on a specific resource",
+            "default": null,
+            "anyOf": [
+                {
+                    "type": "null"
+                },
+                {
+                    "sender": {
+                        "$ref": "resource-details-schema.json"
+                    }
+                },
+                {
+                    "receiver": {
+                        "$ref": "resource-details-schema.json"
```

Comment:
@hrbriggs I assume this is a typo because the actual file you've added is named without an `s` after `detail`?

I'll also take this opportunity to mention that schemas in the specs don't have _-schema_ in the name.
Perhaps also worth being consistent with the specs on lowercase/hyphens/underscores vs CamelCase in the filenames?

Reply:
Are you going to use _resource_details.json_ everywhere else it could be too?

Reply:
Have renamed the files to remove CamelCase (was copying the API file names) but there are schema files with -schema in the name. I was looking at the IS-05 ones I think.

Reply:
You're right, thanks 👍

Reply:
Assume you'll fix the docs and RAML links in another commit??

Reply:
Done